src/ngx_stream_upsync_module.c

```diff
@@ -1155,7 +1155,7 @@ ngx_stream_upsync_consul_parse_json(void *data)
         cJSON *temp1 = cJSON_GetObjectItem(server_next, "Key");
         if (temp1 != NULL && temp1->valuestring != NULL) {
             p = (u_char *)ngx_strrchr(temp1->valuestring, '/');
```

Comment:
(p == NULL) doesnt mean (temp1->valuestring) is null, we should give some message why that is not work, is this better?